docs/prometheus.md

```diff
@@ -56,17 +56,28 @@ sum(node_total_hourly_cost) * 730
 
 > **Note**: Metrics today have both *instance* and *node* labels. The *instance* label will be deprecated in a future version.
```

Comment:
Does this still apply? As in, all node label metrics also can be written with instance?

Reply:
I don't know the state of this, do you @AjayTripathy?

Reply:
Yes, I believe this is still true. You can check quickly by running a prom query and looking for the "instance" tag.